src/pages/components/TabView.qml

```diff
@@ -39,12 +42,15 @@ SilicaGridView {
     delegate: TabItem {
         id: tabItem
 
+        readonly property bool largeLeftMargin: index % columns === 0
+        readonly property bool largeRightMargin: columns > 1 ? index % columns === (columns - 1) : true
```

Comment:
I would rename the properties to something like this:
- "largeLeftMargin" -> "firstColumn"
- "largeRightMargin" -> "lastColumn"
Maybe it's just me, but I find it easier to understand the margin bindings (below) if they were named like this.